docs/migration_guide.md

```diff
@@ -0,0 +1,90 @@
+
+Here's a comprehensive guide to help you through the process.
+
+
+## Before you migrate
+
+Here are some key differences between Percona Everest and PMM DBaaS:
+
+1. Percona Everest has a separate and configurable namespace for running operators and database clusters, whereas PMM/DBaaS uses a default namespace.
+
+2. Percona Everestctl is a tool that helps you install and configure operators and monitoring features, whereas this function was previously handled by PMM.
+
+3. Percona Everest has revamped its backup/restore feature, which means that old backups/restores cannot be used.
+
+
+## Prerequisites
+
+Before getting started with Percona Everest:
+{.power-number}
+
+1. Install [Docker Engine](https://docs.docker.com/engine/install){:target="_blank"} (1.13.0 and higher) with the [Docker compose plugin](https://docs.docker.com/compose/install/){:target="_blank"}.
+
+2. Install [curl](https://everything.curl.dev/get){:target="_blank"}.
+
+3. Install [jq](https://jqlang.github.io/jq/){:target="_blank"}.
+
+4. Set up a publicly accessible Kubernetes cluster. 
+
+    Percona Everest assists with installing all the necessary operators and required packages, but does not currently help with spinning up a publicly accessible Kubernetes cluster.
+
+    We recommend setting up Percona Everest on the Amazon Elastic Kubernetes Service (EKS) or Google Kubernetes Engine (GKE), as Percona Everest may not work as expected on local Kubernetes installations (minikube, kind, k3d, or similar products) due to network issues.
+
+   
+    [Create EKS cluster :material-arrow-right:](quickstart-guide/eks.md){.md-button} [Create GKE cluster :material-arrow-right:](quickstart-guide/gke.md){.md-button}
+
+5. Verify that you have access to the Kubernetes cluster that you want to use with Percona Everest. By default, Percona Everest uses the kubeconfig file available under `~/.kube/config`. 
+
+    To verify access to the Kubernetes cluster, run:
+   
+    ```sh 
+    kubectl get nodes
+    ```
+
+??? example "Expected output"
+    
+    ```{.text .no-copy}
+    NAME                                    STATUS   ROLES    AGE   VERSION
+    gke-<name>-default-pool-75d48bfc-bx8g   Ready    <none>   11h   v1.26.7-gke.500
+    gke-<name>-default-pool-75d48bfc-c2df   Ready    <none>   11h   v1.26.7-gke.500
+    gke-<name>-default-pool-75d48bfc-zl7k   Ready    <none>   11h   v1.26.7-gke.500
+    ```
+
+
+## Migration procedure
+
+1. Set up a publicly accessible Kubernetes cluster.
+2. [Provision](use/db_provision.md) the cluster.
+3. Migrate backup storages from PMM to Percona Everest:
+
+    - Log in to the PMM UI and navigate to the **Backup Storages** section.
+    - Create a [backup](https://docs.percona.com/percona-monitoring-and-management/get-started/backup/index.html).
+    - Log in to Percona Everest and go to the **[Backup Storages](use/backupRestore.md)** section on the UI.
+    - Copy the backup from PMM and paste it to Percona Everest.
```

Comment:
This isn't correct.

> - Log in to the PMM UI and navigate to the **Backup Storages** section.

We should better guide the users to the backup storages.

> - Create a [backup](https://docs.percona.com/percona-monitoring-and-management/get-started/backup/index.html).
> - Log in to Percona Everest and go to the **[Backup Storages](use/backupRestore.md)** section on the UI.
> - Copy the backup from PMM and paste it to Percona Everest.

This is not what we want, we want for the user to take the values available in the backup storages section of PMM and create a backup storage for each of them in everest by following [the prepare a storage location guide](https://docs.percona.com/everest/use/backupRestore.html#prepare-storage-location)